docs_samples/01_python_app_insights_overview.md

```diff
@@ -0,0 +1,239 @@
+python -m pip install opencensus-ext-logging
+python -m pip install opencensus-ext-azure
+python -m pip install opencensus-extension-azure-functions
+python -m pip install opencensus-ext-requests
+python -m pip install pyodbc
+python -m pip install psutil
+
+```
+
+## Configure middleware to track requests
+
+- Browse to the Azure Portal
+- Select your Azure Monitor resource
+- On the **Overview** page, copy the connection string
+- Open the `.env` file and paste it into the `APPLICATIONINSIGHTS_CONNECTION_STRING` value
+
+## Create a simple Python App
+
+- Browse to the Azure Portal.
+- Select your Azure Monitor resource.
+- On the **Overview** page, copy the connection string and save it for later.
+- Create a new python file called `app.py`.
+- Copy the following into it, be sure to replace your Application Insights connection string:
+
+```python
+import logging
+
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+
+logger = logging.getLogger(__name__)
+
+logger.addHandler(AzureLogHandler(connection_string='InstrumentationKey=<your-instrumentation_key-here>'))
+
+logger.warning('Hello, World!')
+```
+
+- Press **F5** to run the file, select **Python file** in the debug configuration window
+- If you get an error about `psutil._psutil_windows` do the following:
+  - Open the `.venv\Lib\site-packages` folder and delete the `psutil` and `psutil-5.9.1.dist-info` folders
+  - Then run the following:
+
+    ```Python
+    python -m pip install --upgrade pip
+    python -m pip install psutil
+    ```
+
+- Switch to the Azure Portal, navigate to your Application Insights resource
+- Under **Monitoring**, select **Logs**
+- Run the following kusto query:
+
+```kusto
+traces
+| sort by timestamp
+| where cloud_RoleName == "app.py"
+```
+
+- You should see the following:
+
+  ![The query is displayed with one result from the above program.](./media/python_simple_app_trace.png "Review the results of the query.")
+
+## Sending trace data
+
+- Open the `./SimpleApps/trace.py` file
+- Press **F5** to run the file
+- Switch to the Azure Portal
+- Browse to your lab resource group
+- Browse to the `python-appinsights-SUFFIX` application insights resource and select it
+- Under **Monitoring**, select **Logs**
+- Run the following Kusto query to see the `traces` sent to application insights:
+
+```kql
+traces
+| sort by timestamp
+| where cloud_RoleName == "trace.py"
+```
+
+- You should see your trace in the **message** column:
+
+  ![The query is displayed with one result from the above program.](./media/python_simple_trace_trace.png "Review the results of the query.")
+
+- For your trace item notice that some of the columns are empty, but others (such as `cloud_RoleName`, `cloud_RoleInstance` and `client_*`) are populated based on the client information from the SDK.
+
+## Logger methods
+
+When working with the [Python logging framework](https://docs.python.org/3/library/logging.html), you should take note of the following methods:
+
```

Comment:
What is this section for? What does it mean for the user to "take note"?

Reply:
removed it...was going to put more content in it but decided to just delete it.